service_container.rst

```diff
@@ -1072,6 +1072,48 @@ If you want to pass the second, you'll need to :ref:`manually wire the service <
     and the automatically loaded service will be passed - by default - when you type-hint
     ``SiteUpdateManager``. That's why creating the alias is a good idea.
 
+Injecting the content of YAML file as argument
+----------------------------------------------
+
+.. versionadded:: 4.3
+    Support for the YAML file notation in YAML and XML was introduced
+    in Symfony 4.3.
+
+You can automatically parse a YAML file and injecting it as an argument by referencing the **absolute path** to the file,
+or the relative path from **your configuration file**:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/services.yaml
+        services:
+            # ...
+
+            App\ServiceWithArrayConfigFromYamlFile:
+                arguments:
+                    - !yaml_file '%kernel.root_dir%/some/file.yaml'
+
+    .. code-block:: xml
+
+        <!-- config/services.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
```

Comment:
Please use https here

Reply:
None of the previous examples use https, sure about that one?

Reply:
https://github.com/symfony/symfony-docs/pull/11122 is on the go, any addition should be secure now.

Reply:
Ok then, fixed!